### PR TITLE
Fix tfjob dashboard  ambassador route

### PR DESCRIPTION
**Pull request by wbuchwalter**
Needed for: https://github.com/kubeflow/tf-operator/pull/441

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/381)

### CODE REVIEW

File: kubeflow/core/tf-job.libsonnet [hunk 1]
```diff
@@ -280,8 +280,8 @@
               "apiVersion: ambassador/v0",
               "kind:  Mapping",
               "name: tfjobs-ui-mapping",
-              "prefix: /tfjobs/ui/",
-              "rewrite: /",
+              "prefix: /tfjobs/",
+              "rewrite: /tfjobs/",
```
Comment (jlewi) [inline kubeflow/core/tf-job.libsonnet]: Can you add a comment explaining  why prefix and rewrite are set the way they are? I would like to understand better what it takes to make Ambassador work. Does the TFJobs UI expect that is rooted at "/"?

Comment (wbuchwalter) [reply kubeflow/core/tf-job.libsonnet]: Please see explanation here: https://github.com/kubeflow/tf-operator/pull/441#discussion_r173168960

Comment (wbuchwalter) [reply kubeflow/core/tf-job.libsonnet]: @jlewi Did you have a chance to look at my explanations?

**Comment by jlewi** [comment]
@wbuchwalter Sorry this slipped my mind again. Make sure you /assign me so it shows up in my PR dashboard
https://k8s-gubernator.appspot.com/pr

**Comment by jlewi** [comment]
presubmit didn't seem to run.
/retest

**Comment by wbuchwalter** [comment]
/assign jlewi
Test fixed, but had to merge master.

**Pull request merged**